data/fcitx5-configtool.sh

```diff
@@ -125,6 +129,13 @@ run_qt() {
     return 1
 }
 
+run_dde() {
+    if which dde-control-center > /dev/null 2>&1; then
+        dbus-send --print-reply --dest=com.deepin.dde.ControlCenter /com/deepin/dde/ControlCenter com.deepin.dde.ControlCenter.ShowPage string:'keyboard/Manage Input Methods' > /dev/null 2>&1 
```

Comment:
Is this page dedicated for fcitx? or it's for deepin in general? What does it looks like?

Also I think it better provides a way to detect such page exists or not.

If dde-control-center can have a flag to be used in a similar way like systemsettings5 above, I'd prefer to use that instead of just dbus-send.